### PR TITLE
Feature/mobileapp 164 carby build instructions

### PR DESCRIPTION
**Pull request by pavelaron**

### CODE REVIEW

File: RevolutionRobotics/Resources/Assets/Images.xcassets/Bitmap_Copy.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "Bitmap_Copy.png",
```
Comment (bmaczak) [inline RevolutionRobotics/Resources/Assets/Images.xcassets/Bitmap_Copy.imageset/Contents.json]: Ennek lehet hasznos lenne valami beszédesebb név

File: RevolutionRobotics/ViewControllers/Menu/MenuViewController.swift [hunk 2]
```diff
@@ -40,12 +40,9 @@ extension MenuViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        let shouldShowTutorial = UserDefaults.standard.bool(forKey: UserDefaults.Keys.shouldShowTutorial)
 
         checkMinVersion(with: { [weak self] in
-            if shouldShowTutorial {
-                self?.showTutorial()
-            }
+            self?.showOnboarding()
```
Comment (bmaczak) [inline RevolutionRobotics/ViewControllers/Menu/MenuViewController.swift]: Shouldn't you check here if the onboarding were already shown?

Comment (bmaczak) [reply RevolutionRobotics/ViewControllers/Menu/MenuViewController.swift]: Nevermind :)

File: RevolutionRobotics/Resources/Assets/Images.xcassets/Bitmap_Copy.imageset/Contents.json (hunk 1 above)
Comment (pavelaron) [reply RevolutionRobotics/Resources/Assets/Images.xcassets/Bitmap_Copy.imageset/Contents.json]: Are we switching to Hungarian :) ? This is just the file's name (as it has been imported from Sympli). In the images.xcassets, it has it's own proper name (bg_main).

File: RevolutionRobotics/ViewControllers/Menu/MenuViewController.swift (hunk 2 above)
Comment (pavelaron) [reply RevolutionRobotics/ViewControllers/Menu/MenuViewController.swift]: Should I rename it to something like `checkOnboarding()` ?

**Pull request merged**